.github/workflows/build.yaml

```diff
@@ -199,11 +199,11 @@ jobs:
       - name: Setup python
         uses: actions/setup-python@v2
         with:
-          python-version: 3.7
+          python-version: 3.6
```

Comment:
Afaik there is no value in using 3.7 in tests. Either 3.6 makes sense (lowest end of the supported range) or 3.9 would be nice here; but not 3.7.